### PR TITLE
fix(push): adjust metrics increment logs

### LINKED ISSUE #1253: Switch push metrics from DataDog to Heka logs
Will help close out https://github.com/mozilla/fxa/issues/144

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #1253

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk @eoger r?

@rfk After this is merged we can hopefully kindly ask whd to create a pie chart / graph in Kibana for us.

File: lib/push.js [hunk 1]
```diff
@@ -46,6 +54,21 @@ module.exports = function (log, db) {
...
+   * Name of the push action
+   */
+  function incrementPushAction(name) {
+    if (name) {
+      log.info({
+        op: LOG_OP_PUSH_TO_DEVICES,
+        name: name
+      })
+    }
+  }
```
Comment (rfk) [inline lib/push.js]: Worth still doing the `log.increment` here as well as `log.info`, for when datadog gets more reliable?

**Comment by rfk** [comment]
LGTM apart from travis bustage, does it need additional fixes for logger methods?

**Comment by eoger** [comment]
👍

**Comment by vladikoff** [comment]
Travis is green and I added the old DataDog events back in. thanks for reviews!

**Pull request merged**